crates/flox/src/commands/channel.rs

```diff
@@ -26,6 +30,12 @@ enum ChannelType {
     Flox,
 }
 
+// This is the `PKGDB` path that we actually use.
+// This is set once and prefers the `PKGDB` env variable, but will use
+// the fallback to the binary available at build time if it is unset.
+pub static PKGDB_BIN: Lazy<String> =
+    Lazy::new(|| env::var("PKGDB").unwrap_or(env!("PKGDB_BIN").to_string()));
```

Comment:
I wonder if we should use `PKGDB_BIN` and `PKGDB_BIN_STATIC` (and then be consistent with that for `NIX_BIN` and `PARSER_UTIL_BIN`)